src/modules/endpoint/selectors/isPeekingRoute.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import type { StateType } from '../../app/StateType'
+import { type InputSelector } from 'reselect'
+
+type SelectPropsType = {routeCity: string}
+
+/**
+ * This fetch corresponds to a peek if the major content city is not equal to the city of the current route.
+ * In this case the fetching and header behaves different. It doesn't fetch resources for example.
+ *
+ * @param state The current state
+ * @param params The params which contain the routeCity
+ * @returns true if the fetch corresponds to a peek
+ */
+const isPeekingRoute: InputSelector<StateType, SelectPropsType, boolean> = (
```

Comment:
Couldn't you just supply peeking when pushing the route?

Reply:
That's how it was before this. It actually is easier this way. We had redundant information in the redux store before this.

Reply:
Allright